docs/user-guide/set-up-from-template.rst

```diff
@@ -184,8 +142,8 @@ This development server auto-reloads, so any time you change the code, the serve
 Next steps
 ==========
 
-Now that you have a working application repository, the next steps are to develop your application's logic and interface, and then deploy it to Roundtable.
+Now that you have a working application repository, the next steps are to develop your application's logic and interface, and then deploy it to Phalanx.
 
 To learn learn more about developing Safir-based applications like yours, refer to the :doc:`guides in this documentation <index>` and the `FastAPI documentation <https://fastapi.tiangolo.com/>`__.
 
-To learn how to deploy your application to Roundtable, see the `Roundtable documentation <https://roundtable.lsst.io>`__.
+To learn how to deploy your application to Phalanx, see the `Phalanx documentation <https://phalanx.lsst.io>`__.
```

Comment:
Perhaps point directly to the developers section of the Phalanx docs: https://phalanx.lsst.io/developers/index.html